GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs

```diff
@@ -189,8 +189,12 @@ public IGTFSFeed GetFeed(int id)
         /// </summary>
         private void RebuildDB()
         {
+            // Alter existing tables for backwards compatibility
+            //  1. add agency_email column to agency
+            if (TableExists("agency") && !ColumnExists("agency", "agency_email")) this.ExecuteNonQuery("ALTER TABLE [agency] ADD [agency_email] TEXT;");
```

Comment:
you could add this at the end of RebuildDB() and then you don't need to check if the table exists, you just need to check if the column doesn't exist. It's just a bit less logic